scte35/state_test.go

```diff
@@ -303,48 +286,34 @@ func TestOutOutIn(t *testing.T) {
 	if len(st.Open()) != 0 {
 		t.Error("Unexpected number of open decriptors found")
 	}
-
-	c, e = st.ProcessDescriptor(close.Descriptors()[1])
-	if e != gots.ErrSCTE35MissingOut {
-		t.Error("Processing second desc of close return unexpected err:", e)
-	}
-	if len(c) != 0 {
-		t.Error("Processing second desc of close did not close outer signal")
-	}
-	if len(st.Open()) != 0 {
-		t.Error("Open signals remain after processing close signal")
-	}
 }
 
 func TestOutOut(t *testing.T) {
 	state := NewState()
-
-	// 0x36 - event_id:0 - seg_num: 0 - seg_expected: 0
-	outSignalBytes, _ := base64.StdEncoding.DecodeString("/DBLAAFztMbuAP/wBQb+AAAAAAA1AjNDVUVJAAAAAH//AACky4AJH1NJR05BTDozR1NOajNnb01sb0FBQUFBQUFBQkFRPT02AADO/OgI")
-	outSignal, err := NewSCTE35(append([]byte{0x0}, outSignalBytes...))
+	// 0x30 - Out signal 1
+	padOpenSignalBytes, _ := base64.StdEncoding.DecodeString("/DBsAAH/7m1eAAKQBQb+sX6o+wBWAlRDVUVJAAAAJ3//AAApMuANQAwOQU1DTiBMMDAxMjM0NTYJCFBPOjEyMzQ1DiRiY2IxZGQ4ZS1kMzIzLTQ1ODktOWQ3OC1hM2QxMzYyYWJiYjYwAQGtc8xr")
+	padOpenSignal, err := NewSCTE35(append([]byte{0x0}, padOpenSignalBytes...))
 	if err != nil {
 		t.Errorf("Error creating SCTE-35 signal: %s", err.Error())
 	}
 
-	closed, err := state.ProcessDescriptor(outSignal.Descriptors()[0])
+	_, err = state.ProcessDescriptor(padOpenSignal.Descriptors()[0])
 	if err != nil {
-		t.Errorf("ProcessDescriptor returned an error: %s", err.Error())
-	}
-	if len(closed) != 0 {
-		t.Errorf("No events should have been closed (%d were)", len(closed))
+		t.Error("ProcessDescriptor of out returned unexpected err:", err)
 	}
 	if len(state.Open()) != 1 {
-		t.Errorf("There should be one open signal (%d)", len(state.Open()))
+		t.Error("Unexpected number of open signals after padOpenSignal processed.")
```

Comment:
Can we add back the `len(state.Open())`? I think it's useful information for an error message.